pihole

```diff
@@ -269,10 +269,24 @@ piholeCheckoutFunc() {
 }
 
 tricorderFunc() {
+  if [ ! -p "/dev/stdin" ]; then
+    echo "Please do not call Tricorder directly."
+    exit 1
+  fi
+
   if command -v openssl &> /dev/null; then
     openssl s_client -quiet -connect tricorder.pi-hole.net:9998 2> /dev/null < /dev/stdin
   else
-    nc tricorder.pi-hole.net 9999 < /dev/stdin
+    echo "The debug log will be transmitted insecurely via plain-text"
```

Comment:
You're absolutely right! Check the most recent version, and let me know what you think of the suggested wording.